proto/backend.proto

```diff
@@ -38,6 +38,7 @@ message DataSourceInstanceSettings {
   // The API version when the settings were saved.
   // NOTE: this may be an older version than the current apiVersion
   string apiVersion = 12;
+  bool system = 13;
```

Comment:
I get why you may need this in the UI... but I do not get why the protobuf needs it